test/models/InstanceTest.scala

```diff
@@ -0,0 +1,40 @@
+package models
+
+import org.scalatest.{FreeSpec, Matchers}
+import util.Fixtures._
+
+class InstanceTest extends FreeSpec with Matchers {
+
+  val i1 = emptyInstance("i1").copy(stack = Some("stack1"), stage = Some("stage1"), app = List("app1"))
+
+  "Instance" - {
+    "should belong to SSA with same stack name" in {
```

Comment:
FYI, you can nest blocks using FreeSpec, which would make these messages less boiler-platey.

```scala
"Stack instance" - {
  "belongs to SSA" - {
    "with same stack name" in {
      val stack = SSA(stack = i1.stack)
      i1.belongsToSSA(stack) should be (true)
    }
    "with same stack and stage" in {
      val stack = SSA(stack = i1.stack, stage = i1.stage)
      i1.belongsToSSA(stack) should be (true)
    }
  }
  "does not belong to SSA" - {
    // etc
```

It's also extremely helpful for sharing test setup using block scope (not relevant in this case but worth keeping in mind). This is why I favour FreeSpec, it's incredibly flexible and powerful and doesn't depend on magic context like the "**it** should" pattern in many other matcher choices.